src/internal/svg/disconnectSvg.tsx

```diff
@@ -1,22 +1,20 @@
-import { fill } from '../../styles/theme';
-
 export const disconnectSvg = (
   <svg
     role="img"
     aria-label="ock-disconnect-svg"
-    width="100%"
-    height="100%"
-    viewBox="0 0 20 20"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
     <path
-      d="M10.9735 2.69696L10.9735 4.52272L5.49622 4.52272L5.49622 15.4773L10.9735 15.4773L10.9735 17.303L3.67046 17.303L3.67046 2.69696L10.9735 2.69696Z"
-      className={fill.defaultReverse}
+      d="M8.77876 2.15757L8.77876 3.61818L4.39698 3.61818L4.39698 12.3818L8.77876 12.3818L8.77876 13.8424L2.93637 13.8424L2.93637 2.15757L8.77876 2.15757Z"
+      fill="#0A0B0D"
```

Comment:
Can you check with @kyhyco in making sure we still support the theming here.

Reply:
Yeah, after the most recent rebase I can't get my BaseNames svg to render